packages/salesforcedx-vscode-apex/src/apexErrorHandler.ts

```diff
@@ -0,0 +1,62 @@
+import { EventEmitter } from 'events';
+import {
+  CloseAction,
+  ErrorAction,
+  ErrorHandler,
+  LanguageClient,
```

Comment:
@lukeis There are unused imports in this file. `Cmd-Shift-P` then `Organize Imports` will remove the unused and make sure they are ordered correctly. I should of done this before pushing.